### PR TITLE
MGDAPI-5651  adding upgrade scenario verification required

### PR DESCRIPTION
**Pull request by laurafitzgerald**
# Issue link
https://issues.redhat.com/browse/MGDAPI-5651

# What
Updating the git pull template to include requirement for upgrade verification OR a reason included 

# Verification steps
No verification steps required here as it needs to be merged before it gets seen. An eyeball of the changes to confirm that the are as expected is sufficient.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,6 +5,6 @@
 <!-- describe a summary of the change, add any additional motivation and context as needed -->
 
 # Verification steps
-<!-- describe verification steps with sufficient level of detail -->
-<!-- take into consideration upgrade scenario, RHMI 2.x, etc. -->
+<!-- Add thorough verification steps with sufficient level of detail for those without context to verify the change-->
+<!-- AND Add thorough upgrade verification OR include a reason as to why it is not required -->
```
Comment (Fiona-Waters) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Do you want steps here?
```suggestion
<!-- AND Add thorough upgrade verification steps OR include a reason as to why it is not required -->
```
The rest of it looks great to me.

Comment (laurafitzgerald) [reply .github/PULL_REQUEST_TEMPLATE.md]: Updated.

**Pull request merged**